packages/main/src/TabInOverflow.hbs

```diff
@@ -19,3 +20,9 @@
 		{{/if}}
 	</div>
 </ui5-li-custom>
+{{#if this.subTabs}}

```

Comment:
Removing this guard should still be ok. {{#each..}} should not print anything, if the array subtabs is emply. Also try without `this.` as well. HBS docs suggest that would work.